packages/sync-core/src/lib/protocol.ts

```diff
@@ -9,7 +9,7 @@ export function getTlsyncProtocolVersion() {
 }
 
 /**
- * @internal
+ * @public
```

Comment:
can we keep the protocol and message format stuff internal? I'd rather make the socket wrapper type less safe. I don't think it really benefits from knowing the message format anyway.